### PR TITLE
Add database/config GET and database/config Patch

### PR DESCRIPTION
**Pull request by ebergstromDO**
This PR adds the database config commands to the database part of the API. I would appreciate a fairly in depth review as I'm unsure of it.

### CODE REVIEW

File: specification/resources/databases/get_database_config.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+operationId: get_database_config
+
+summary: Retrieve Configuration for an Existing Database
+
+description: |
+  To show configuration parameters for an existing database cluster, send a GET request to
+  `/v2/databases/$DATABASE_ID/config`.
+
+  Note: Database config is not yet supported for Redis and Postgres clusters.
```
Comment (ChiefMateStarbuck) [inline specification/resources/databases/get_database_config.yml:9]:
```suggestion
  **Note:** Database config is not yet supported for Redis and Postgres clusters.
```

It looks like the word `NOTE:` is bolded in the rest of the Database files.

File: specification/resources/databases/patch_database_config.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+operationId: patch_database_config
+
+summary: Change configuration values
+
+description: |
+  To update configuration for an existing database cluster, send a PATCH request to
+  `/v2/databases/$DATABASE_ID/config`.
+
+  Note: Database configuration is not supported for Redis and Postgres clusters.
```
Comment (ChiefMateStarbuck) [inline specification/resources/databases/patch_database_config.yml:9]:
```suggestion
  **Note:** Database configuration is not supported for Redis and Postgres clusters.
```

**Review by ChiefMateStarbuck — CHANGES_REQUESTED** [review]
Very good work. Thank you. Although an approval from another DATABASE team member is not necessary, it is certainly helpful.

**Review by rpmoore — APPROVED** [review]

**Comment by bentranter** [comment]
Closing in favour of #590

**Pull request closed without merging**